src/DynamicRupture/FrictionLaws/GpuImpl/AgingLaw.h

```diff
@@ -0,0 +1,44 @@
+#ifndef SEISSOL_GPU_AGINGLAW_H
+#define SEISSOL_GPU_AGINGLAW_H
+
+#include "DynamicRupture/FrictionLaws/GpuImpl/SlowVelocityWeakeningLaw.h"
+
+namespace seissol::dr::friction_law::gpu {
+
+/**
+ * This class was not tested and compared to the Fortran FL3. Since FL3 initialization did not work
+ * properly on the Master Branch. This class is also less optimized. It was left in here to have a
+ * reference of how it could be implemented.
+ */
+template <class TPMethod>
+class AgingLaw : public SlowVelocityWeakeningLaw<AgingLaw<TPMethod>, TPMethod> {
+  public:
+  using SlowVelocityWeakeningLaw<AgingLaw<TPMethod>, TPMethod>::SlowVelocityWeakeningLaw;
+  using SlowVelocityWeakeningLaw<AgingLaw<TPMethod>, TPMethod>::copyLtsTreeToLocal;
+
+  void updateStateVariableImpl(double timeIncrement) {
```

Comment:
Does `Impl` stand for implicit or implementation? Please change the name to be not ambiguous.

Reply:
Hi. I've seen many times when people use `Impl` as an abbreviation of `implementation`. Anyways, I renamed it back to `updateStateVariable` as it was done in the CPU version.

I don't know why but at the beginning my linter showed me an issue with respect to overloading of this method. Not it doesn't complain anymore.